src/plugins/imagery/components/Compass/CompassRose.vue

```diff
@@ -177,12 +185,28 @@ export default {
         }
     },
     computed: {
-        north() {
-            return this.lockCompass ? rotate(-this.cameraPan) : 0;
+        compassRoseSizing() {
+            let compassRoseSizing = '';
+            if (this.sizedImageWidth < 300) {
+                compassRoseSizing = '--rose-small --rose-min';
+            }
+
+            if (this.sizedImageWidth < 500) {
+                compassRoseSizing = '--rose-small';
+            }
+
+            if (this.sizedImageWidth > 1000) {
+                compassRoseSizing = '--rose-max';
+            }
+
+            return compassRoseSizing;
```

Comment:
I remember we talked about this before, I think we can clean this up a bit. we had talked about returning right out of the if statements right? I think that might be the way to go here OR you could just chain these if's in to if else statements. that way we're not doing erroneous checks. Or a combo... something like:


``` Javascript
compassRoseSizingClasses() {
        if (this.sizedImageWidth < 300) {
            return '--rose-small --rose-min';
        } else if (this.sizedImageWidth < 500) {
            return '--rose-small';
        } else if (this.sizedImageWidth > 1000) {
            return '--rose-max';
        }
        
        return '';
}
```

Reply:
Ugh, looking at what I wrote now and it's terrible. Good suggestion, fixed.